features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListener.kt

```diff
@@ -292,12 +319,7 @@ internal class JankStatsActivityLifecycleListener(
                 " shouldn't happen."
         internal const val JANK_STATS_TRACKING_DISABLE_ERROR =
             "Failed to disable JankStats tracking"
-
-        private val ONE_SECOND_NS: Double = TimeUnit.SECONDS.toNanos(1).toDouble()
-
-        private const val MIN_FPS: Double = 1.0
-        private const val MAX_FPS: Double = 60.0
         private const val SIXTY_FPS: Double = 60.0
-        private const val SIXTEEN_MS_NS: Long = 16666666
+        private const val TRUE = 1L
```

Comment:
Maybe we can name it something like `IS_FIRST_DRAW_FRAME` to be less generic?

```suggestion
        private const val IS_FIRST_DRAW_FRAME = 1L
```